tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: add-fbc-contribution
   labels:
-    app.kubernetes.io/version: "1.0.0"
+    app.kubernetes.io/version: "1.0.1"
```

Comment:
Even though the change is small, I think this should be 1.1.0. My understanding is the final number is only bumped for backwards compatible bugfixes

Reply:
fixed